Add :source field to data

ondenman opened the pull request:
The scraper was not capturing the source of the member data. This is being added because our existing data contains the source of the data for each member.

## [Scraper Change checklist](https://github.com/everypolitician/everypolitician/wiki/Scraper-Change-checklist)
* [x] 1. scraper is on Morph.io under the "everypolitician-scrapers" group?
* [x] 2. scraper's GitHub "Website" link points at morph.io page?
* [x] 3. scraper is set to auto-run?
* [x] 4. scraper is archiving?
* [x] 5. legislature has a scraper webhook set?

ondenman committed:
Add :source field to data

tmtmtmtm commented on scraper.rb:
```diff
@@ -17,6 +17,7 @@
   noko = Nokogiri::HTML(open(base_url + url).read)
   TermPage.new(noko).members.each do |mem|
     mem[:term] = term
+    mem[:source] = URI.join('https://en.wikipedia.org/wiki/', url).to_s
```
You're now generating this URL twice, and differently each time — once with a variable that contains the base URL, and once with it hardcoded; then once with `URI.join`, and once with `base_url + url`.

ondenman committed:
Generate absolute URL once only

Generate absolute URL before we loop over the terms instead of
generating it twice within the loop.

tmtmtmtm commented on scraper.rb:
```diff
@@ -11,12 +11,13 @@
+  noko = Nokogiri::HTML(open(url).read)
   TermPage.new(noko).members.each do |mem|
     mem[:term] = term
+    mem[:source] = URI.join(url).to_s
```
isn't `URI.join(url).to_s` just `url`?
